### DISCUSSION TITLE
add Matrox Impression emulation

### DISCUSSION THREAD

**Discussion by WhiteMagicRaven** [86Box/86Box#4555] [Ideas]
Matrox Impression is released before Millennium and Mystique.
I believe its Millennium is Impression evolution, so impression have least features.
Adding emulation of this is good idea to preserve 4 exclusive games, and couple demos. Worth for GAME PRESERVATION ))

i have bios, have videos, and have video card itself, i saw there is datasheet https://www.vgamuseum.info/index.php/cards/item/203-matrox-is-athena-mga-impression-plus

**Comment by Torinde** [comment]
Related: https://github.com/joncampbell123/dosbox-x/issues/291
Duplicate: https://github.com/86Box/86Box/discussions/1933#discussioncomment-1801333

**Comment by WhiteMagicRaven** [comment]
also some partial API calls maybe exposed here https://www.vogons.org/viewtopic.php?t=94807

**Comment by WhiteMagicRaven** [comment]
Also lots of Matrox documentation 
https://bitsavers.trailing-edge.com/components/matrox/_dataSheets/

**Comment by a-1103** [comment]
Why 86box Matrox emulation does not use latest BIOS versions?
ftp://ftp.matrox.com/pub/mga/bios/setup351.exe
```
  Mystique -----------------------BIOS version 1.8.37-----858-8
  Mystique RRunner----------------BIOS version 1.8.14-----861-8
  Millennium----------------------BIOS version 3.0.60-----838-20
  MillenniumII PCI ---------------BIOS version 1.5.24-----859-5
  MillenniumII AGP----------------BIOS version 1.4.14-----865-4
  Millennium II PCI RRunner-------BIOS version 1.4.03-----876-4
  Millennium II AGP RRunner-------BIOS version 1.3.03-----877-3
  MGA G100 AGP--------------------BIOS version 1.6.22-----874-6
  MGA G100 PCI--------------------BIOS version 1.2.09-----873-2
  MGA G100 TVO AGP----------------BIOS Version 1.5.01-----874-5TVO
```
